src/main/crossover.js

```diff
@@ -181,6 +181,7 @@ const lockWindow = ( lock, targetWindow = windows.win ) => {
 	const hideOnMouse = Number.parseInt( preferences.value( 'actions.hideOnMouse' ), 10 )
 	const hideOnKey = preferences.value( 'actions.hideOnKey' )
 	const tilt = checkboxTrue( preferences.value( 'actions.tiltEnable' ), 'tiltEnable' )
+	const resizeOnADS = Number.parseInt( preferences.value( 'actions.resizeOnADS' ), 10 )
```

Comment:
Since we changed to `off`, `press`, `hold`, this can just be `preferences.value( 'actions.resizeOnADS' )`